ClassicLootManager/Modules/Rosters/RosterManager/RosterManager.lua

```diff
@@ -347,6 +347,20 @@ function RosterManager:Initialize()
             roster:GetCalculator():SetExpvar(entry:expvar())
         end))
 
+    CLM.MODULES.LedgerManager:RegisterEntryType(
+            CLM.MODELS.LEDGER.ROSTER.DynamicItemValueCustomExpression,
+            (function(entry)
+                LOG:TraceAndCount("mutator(DynamicItemValueCustomExpression)")
+                local rosterUid = entry:rosterUid()
+
+                local roster = self:GetRosterByUid(rosterUid)
+                if not roster then
+                    LOG:Debug("Updating non-existent roster [%s]", rosterUid)
+                    return
+                end
+                roster:GetCalculator():SetCustomExpression(entry:customExpression())
```

Comment:
Validate the expression again before setting it; Error out (`LOG:Error`) if it's invalid